core/opendaq/reader/include/opendaq/multi_reader.h

```diff
@@ -55,8 +56,10 @@ DECLARE_OPENDAQ_INTERFACE(IMultiReader, ISampleReader)
      * available the parameter value is set to the actual amount and only the available
      * samples are returned. The rest of the buffer is not modified or cleared.
      * @param timeoutMs The maximum amount of time in milliseconds to wait for the requested amount of samples before returning.
+     * @param[out] status: The reading status indicates whether the reading process was successful or if an event packet was encountered while processing. 
+     * If an event packet was encountered, the reader can still convert data. Currently returns nothing
      */
-    virtual ErrCode INTERFACE_FUNC read(void* samples, SizeT* count, SizeT timeoutMs = 0) = 0;
+    virtual ErrCode INTERFACE_FUNC read(void* samples, SizeT* count, SizeT timeoutMs = 0, IReaderStatus** status = nullptr) = 0;
```

Comment:
To prevent API changes in the future - should we just create an IMultiReaderStatus that inherits IReaderStatus but is blank?

Or maybe it does not even need to inherit IReaderStatus.